### PR TITLE
add recipe for jupyter_nbextensions_configurator

### PR DESCRIPTION
**Pull request by jcb91**

### CODE REVIEW

**Comment by jcb91** [comment]
🏁

File: recipes/jupyter_nbextensions_configurator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - jupyter_core
+    - nbconvert
+    - notebook >=4.0
+    - pyyaml
+    - setuptools
```
Comment (ocefpaf) [inline recipes/jupyter_nbextensions_configurator/meta.yaml]: Do you need `setuptools` at run time? Maybe something imports `pkg_resources`?

File: recipes/jupyter_nbextensions_configurator/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  license_file: LICENSE.txt
+  license: BSD 3-clause
+
+extra:
+  recipe-maintainers:
+    - jcb91
+
+# See
+# http://docs.continuum.io/conda/build.html for
+# more information about meta.yaml
```
Comment (ocefpaf) [inline recipes/jupyter_nbextensions_configurator/meta.yaml]: Please remove this comments before we merge this.

**Comment by ocefpaf** [comment]
@jcb91 Looks good to go. Just some minor comments and we can merge this.

File: recipes/jupyter_nbextensions_configurator/meta.yaml (hunk 1 above)
Comment (jcb91) [inline recipes/jupyter_nbextensions_configurator/meta.yaml]: exactly, a setuptools-created script imports `pkg_resources`

File: recipes/jupyter_nbextensions_configurator/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+{% set version = '0.1.1' %}
+{% set md5 = 'c53ee4136529b915811cf0925154a291' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/dc/4f/c08c6396ebd5cc8bf12a38892e52a7177abfd2e09c279ac2174d559fbc3f/{{ name }}-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/jupyter_nbextensions_configurator/meta.yaml]: Could we please change this URL as shown below?

```
https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment (jcb91) [inline recipes/jupyter_nbextensions_configurator/meta.yaml]: Sure. I tried that before, but it didn't work for me. I guess I just didn't wait long enough for `pypi.io` to pick up my changes from `pypi.python.org`. Will push an update now...

Comment (jakirkham) [inline recipes/jupyter_nbextensions_configurator/meta.yaml]: Yeah, I think there is some time lag between pushing a release and being able to download it via pypi.io. We have seen it elsewhere too.

**Comment by jakirkham** [comment]
Seems the AppVeyor webhook had an issue firing. Will trigger manually from the AppVeyor side of things.

**Comment by jakirkham** [comment]
AppVeyor has now started.

File: recipes/jupyter_nbextensions_configurator/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipes/jupyter_nbextensions_configurator/meta.yaml]: Appears this is now working.

**Comment by jakirkham** [comment]
Thanks @jcb91.

**Comment by jakirkham** [comment]
As this is your first time signing up as a maintainer, @jcb91, I would like to explain the process a bit. Now that your GitHub handle is listed here, an automated script will come by at some point and notice that you are not currently part of the `jupyter_nbextensions_configurator` teams. It will attempt to add you to the `jupyter_nbextensions_configurator` teams so you have rights on this repo. However, as you are not in conda-forge yet, this will trigger GitHub to send you an email, which invites you to join conda-forge. Once accepted you will be added to the aforementioned teams and a general team ( `all-members` ). From now on, anything you add yourself to or others add you to (with your expressed permission) will result in you automatically being added to those teams by this automation script.

Regarding making changes to the recipe, please proceed with the normal workflow one would with any repository. Please fork the repository, make changes to your fork, and PR them back. This is important to follow exactly as all branches on the original repository will automatically deploy anything they build.

If you have and questions or concerns, please let us know. Welcome to conda-forge. 😄

**Comment by jakirkham** [comment]
Oops, missed the most important part. 😳 The feedstock for this recipe can be found here ( https://github.com/conda-forge/jupyter_nbextensions_configurator-feedstock ).

**Comment by jcb91** [comment]
Awesome, sounds great! This whole forge business is dauntingly impressive, I like it!

>  all branches on the original repository will automatically deploy anything they build

the 'original' in this case being the feedstock, right?

**Comment by jakirkham** [comment]
Glad you like it. 😄 @pelson did the hardwork of getting it up and running. The rest of have just been improving on the model ever since.

> >  all branches on the original repository will automatically deploy anything they build

> the 'original' in this case being the feedstock, right?

Correct.

This is mainly useful if one wants to support multiple versions at a time. In practice, I think there are a few use cases for it, but not as many as one might think.

There have been discussions about tweaking this model. See this issue ( https://github.com/conda-forge/conda-forge.github.io/issues/87 ). Though I have found that communicating the information that I have to you avoids most problems out of the gate. So, instead I have been more interested in doing that and trying to potentially automate it too to avoid missing important bits. See this issue ( https://github.com/conda-forge/conda-forge.github.io/issues/149 )

**Pull request merged**